scripts/monkey-test.sh

```
@@ -0,0 +1,72 @@
+# Ensure that report is generated
+if [ ! -f ast-metrics-report.json ]; then
+    echo "Report not generated"
+    exit 1
+else 
+    echo "Report generated"
+fi
+
+
+# Count number of analyzed files
+# | **PHP** | 122.0 K | 🟢 112 | 1.21 | 12 |
+line=$(cat build/report.md |grep '**PHP**'|head -n 1)
```

Shellcheck (0.10.0)

[warning] Grep uses regex, but this looks like a glob.

(SC2063)